### PR TITLE
Added filename attr to RpmUnit, nsvca property to ModulemdUnit

### PR DESCRIPTION
**Pull request by rbikar**
Added filename attribute to RpmUnit and nsvca property to
ModulemdUnit class.

Tests updated.

### CODE REVIEW

**Comment by rbikar** [comment]
@rohanpm @rajulkumar 
Do you know how to get rid of continuous-integration/travis-ci check?

File: pubtools/pulplib/_impl/model/unit/rpm.py [hunk 1]
```diff
@@ -54,6 +54,12 @@ class RpmUnit(Unit):
         :meth:`~pubtools.pulplib.Repository.signing_keys`
     """
 
+    filename = pulp_attrib(default=None, type=str, pulp_field="filename")
+    """
+    Filename of this RPM.
```
Comment (rohanpm) [inline pubtools/pulplib/_impl/model/unit/rpm.py:59]: Minor - there ought to be a blank line after this.

It's like git commit messages, the first line will be used as a summary in some places.

**Review by rohanpm — DISMISSED** [review]

**Comment by rohanpm** [comment]
> @rohanpm @rajulkumar
> Do you know how to get rid of continuous-integration/travis-ci check?

It's in the branch protection rules at https://github.com/release-engineering/pubtools-pulplib/settings/branches

Checks needed to be updated for travis-ci.org => .com switch.  I thought that I had already done that for this repo, but I guess not.

**Comment by rohanpm** [comment]
> > @rohanpm @rajulkumar
> > Do you know how to get rid of continuous-integration/travis-ci check?
> 
> It's in the branch protection rules at https://github.com/release-engineering/pubtools-pulplib/settings/branches
> 
> Checks needed to be updated for travis-ci.org => .com switch. I thought that I had already done that for this repo, but I guess not.

Fixed now.

**Comment by rbikar** [comment]
I changed only docstring, moving forward with merge.

**Pull request merged**